releasenotes/notes/ansible-requirements-bump-89313038efba83b3.yaml

```diff
@@ -0,0 +1,16 @@
+---
+features:
+  - |
+    Ansible dependencies have been bumped to the latest available
+    versions. This includes:
+
+    * pulp.squeezer - 0.0.13 -> 0.1.1
+    * stackhpc.vxlan - unpinned -> 1.1.0
+    * ansible-lockdown.rhel9_cis - 1.3.1 -> 2.0.1
+    * geerlingguy.pip - 2.2.0 -> 3.1.0
+    * monolithprojects.github_actions_runner - 1.18.5 -> 1.25.1
+    * geerlingguy.docker - unpinned -> stackhpc/7.0.1.1
+    * ansible-modules-hashivault - 5.2.1 -> 5.3.0
+
+    ansible-lockdown.ubuntu22_cis has been replaced with
+    ansible-lockdown.ubuntu24_cis, which is pinned to 1.0.1
```

Comment:
Missing period at the end of the sentence and missing EOL at the end of file. I would also sprinkle some double backticks but that's just me 😄

Reply:
Added, added, sprinkled